appveyor.yml

```diff
@@ -0,0 +1,21 @@
+branches:
+  only:
+    - master
+
+version: '1.0.{build}'
+
+install:
+  - choco install dart-sdk-dev
```

Comment:
I don't thing chocolatey gets updated very often but that is probably ok? I think it's the easiest way to install so the trade-off is probably worth it.

Reply:
Yeah. If we end up needing an edge version we can swap it.